src/tooltip.jsx

```diff
@@ -57,6 +57,7 @@ let Tooltip = React.createClass({
         borderRadius: 2,
         userSelect: 'none',
         opacity: 0,
+        zIndex: 99,
```

Comment:
One strategie could be to have all the zIndex of all element at the same place. The theme could fit this role perfectly